Cargo.toml

```diff
@@ -25,10 +25,6 @@ members = [
     "crypto/crypto",
     "crypto/crypto_derive",
     "crypto/secret_service",
-    "execution/execution_client",
-    "execution/execution_proto",
-    "execution/execution_service",
-    "execution/execution_tests",
```

Comment:
`execution_tests` should not be removed.

Reply:
I will move `storage_integration_test.rs` inside `executor`. So it doesn't need to stay in main `Cargo.toml`. The service test is not needed since the service itself doesn't exist anymore.

Reply:
Should we move it in this diff? Otherwise the test is not being run.